lib/media/streaming_engine.js

```diff
@@ -1330,9 +1330,10 @@ shaka.media.StreamingEngine = class {
               stream.mimeType == 'audio/mp4';
       const isReadableStreamSupported = window.ReadableStream;
       // Enable MP4 low latency streaming with ReadableStream chunked data.
-      // And only for DASH.
+      // And only for DASH and HLS with byterange optimization.
       if (this.config_.lowLatencyMode && isReadableStreamSupported && isMP4 &&
-          this.manifest_.type != shaka.media.ManifestParser.HLS) {
+          (this.manifest_.type != shaka.media.ManifestParser.HLS ||
```

Comment:
Say, it occurs to me. This assumes that a manifest that isn't HLS is DASH, based on the comment above.
But we now have the constant of `shaka.media.ManifestParser.MSS`. Is this behavior supported by MSS?

Reply:
Yes, this would work for MSS live (in the future)